### PR TITLE
do not pollute root directory with chain dumps

### PR DESCRIPTION
**Pull request by clangenb**

### CODE REVIEW

**Comment by brenzi** [comment]
these are dumped by polkadot-launch and still need to be ignored:
rococo-local*
kusama-local*
polkadot-local*

File: .gitignore [hunk 1]
```diff
@@ -9,12 +9,7 @@
 *.log
 
 # autogenerated chain_specs
+chain_dumps/
```
Comment (brenzi) [inline .gitignore:12]: I suggest to call this `spec_dumps` and you need to add the empty directory too or this will fail

**Review by brenzi — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (clangenb) [reply .gitignore:12]: I don't quite agree, it is not only the chain_specs. The `.state` and `.wasm` files are also contained.

What do you mean by 'adding the empty directory'? 
The dump script runs `mkdir -p` to make sure the directory exists, I don't think we need to check this in.

Comment (brenzi) [reply .gitignore:12]: missed that. sorry. can live with chain

**Pull request merged**